### ISSUE TITLE
Wire connecting back

### ISSUE THREAD

**Issue by dpeter99** [amadornes/SCM-Localization-Issues#181]
![2016-11-10_15 39 53](https://cloud.githubusercontent.com/assets/7725817/20180770/16080898-a75c-11e6-9a1d-c653b7c60937.png)

![2016-11-10_15 39 57](https://cloud.githubusercontent.com/assets/7725817/20180769/15ebf27a-a75c-11e6-8cf5-59d49a9e97e3.png)

Only placed the impregnated stick

**Comment by amadornes** [comment]
Yep, can confirm this is a bug. I completely forgot to fix it in 1.0.3!
